schemas/workflow_runs/workflow_run.yaml

```diff
@@ -60,6 +71,14 @@ properties:
         description: Date and time when the link will expire.
       language:
         type: string
+        enum:
```

Comment:
Are these the only languages supported?

Reply:
Yeah, that's according the [documentation](https://documentation.onfido.com/#create-workflow-run-link-object):

> Supported languages
> 
> CODE | LANGUAGE
> -- | --
> en_US | English (default)
> de_DE | German
> es_ES | Spanish
> fr_FR | French
> it_IT | Italian
> pt_PT | Portuguese
> nl_NL | Dutch
>

Reply:
Ok, the team needs to be aware to update this if they add more languages then

Reply:
this can also be moved to a separate file, so that it's edited just once, in case this list gets used anywhere else.